code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -95,6 +95,16 @@
 		evolve()
 		return
 
+/mob/living/simple_animal/mouse/ClickOn(atom/A, params)
```

Comment:
Why use ClickOn? Whoever told you to do that is an idiot.
Anyway, if you really insist on it you should use a proximity check so they can’t eat cheese from 7 tiles away. 

I still recommend you just use AttackingTarget like in corgi code for eating snacks

Reply:
Is `ClickOn` still needed with `AttackingTarget` ?

Reply:
SimpleAnimal's don't have AttackingTarget, that's only on Hostile Simplemobs.

Reply:
`/mob/living/simple_animal/UnarmedAttack(atom/A, proximity)`
What about this?